packages/ui/src/ui/store/reducers/tablet_cell_bundles/index.ts

```diff
@@ -175,6 +141,40 @@ export interface OrchidBundles {
     [bundleName: string]: OrchidBundlesData;
 }
 
+export interface MemoryLimits {
```

Comment:
I don't quite understand the purpose of moving these interfaces in the same file